frontend/src/metabase/search/components/filters/CreatedByFilter/CreatedByContent.styled.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from "@emotion/styled";
+import { Stack } from "metabase/ui";
+
+export const CreatedByContainer = styled(Stack)`
+  overflow: hidden;
+`;
+
+export const CreatedByContentContainer = styled(Stack)`
+  overflow-y: auto;
+
+  ::-webkit-scrollbar {
+    display: none;
+  }
```

Comment:
> replicate the lack of scrollbar for macOS users

This is an [OS setting](https://www.macrumors.com/how-to/make-scroll-bars-always-visible/), the app should respect it rather than override it.

So what's the problem here - is the scrollbar always shown without this CSS on macOS?

Reply:
Yeah, I get a really ugly scrollbar on my side

Reply:
Ok maybe not.

Reply:
Should we test it on windows to see if the scrollbar break the layout?
@oisincoveney could it be that you connected/disconnected a mouse?
Afaik by default scrollbars visibility depends on that

Reply:
YES! I was wondering what was going on, it's a mouse thing

![image](https://github.com/metabase/metabase/assets/25306947/ce3a1f1d-1198-4892-87da-07aedb94d719)

Reply:
The scrollbar doesn't break the layout at least